scripts/test_tf2_execution.py

```diff
@@ -238,10 +251,25 @@ def test_checkpointed(train_ds, test_ds, solver, epochs=1):
     ) as f:
         json.dump(data, f)
 
+def listify(res):
+    return np.array(res).tolist()
+@contextlib.contextmanager
+def options(options):
+    old_opts = tf.config.optimizer.get_experimental_options()
+    tf.config.optimizer.set_experimental_options(options)
+    try:
+        yield
+    finally:
+        tf.config.optimizer.set_experimental_options(old_opts)
 
 if __name__ == "__main__":
```

Comment:
I left old code sitting here but there should be no `__main__` calls in unit tests. Can you take it out, or move it to a file in the `scripts/` directory?

Reply:
Yup, this is no longer neede.